src/commands/deploy.rs

```diff
@@ -93,6 +93,10 @@ pub struct DeployCommand {
         env = DEPLOYMENT_ENV_NAME_ENV
     )]
     pub deployment_env_id: Option<String>,
+
+    /// Pass a key/value (key=value) to all components of the application.
+    #[clap(long, parse(try_from_str = parse_kv))]
+    pub key_values: Vec<(String, String)>,
```

Comment:
The code assumes we return a vector, but parse_kv only extracts a single KV.

Is there a risk by allowing any key and any value with an equal sign as the separator? If we eventually change parse_kv to parse KV pairs separated by commas, would that break compatibility?

This would be allowed in its current form: `”greeting=Hello,World”`

Reply:
I think the `Vec` type means that `clap` will allow multiple instances of the argument, each containing one pair.  This is how it works for `spin up --env` which looks pretty similar.

Reply:
It's a direct lift from `spin up --env`

Reply:
Ah. That makes much more sense. Thank you.